packages/bff/src/model/domain/apiConverter.ts

```diff
@@ -0,0 +1,95 @@
+import {
+  InstitutionResource,
+  ProductResource,
+  UserResource,
+} from "pagopa-interop-selfcare-v2-client";
+import { P, match } from "ts-pattern";
+import {
+  ApiSelfcareInstitution,
+  BffApiSelfcareProduct,
+  BffApiSelfcareUser,
+} from "../api/bffTypes.js";
+import { selfcareEntityNotFilled } from "./errors.js";
+
+export const toApiSelfcareInstitution = (
+  input: InstitutionResource
+): ApiSelfcareInstitution =>
+  match(input)
+    .with(
+      {
+        id: P.not(P.nullish),
+        description: P.not(P.nullish),
+        userProductRoles: P.not(P.nullish),
```

Comment:
```suggestion
        id: P.nonNullable,
        description: P.nonNullable,
        userProductRoles: P.nonNullable,
```
We can use the `P.nonNullable` instead of negating the `nullish`, you can do the same for the following occurences